core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java

```diff
@@ -69,7 +69,13 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   @Override
   public void initialize(BatchRuntimeContext context) throws Exception {
     super.initialize(context);
-    Schema outputSchema = Schema.parseJson(tableSinkConfig.getSchemaStr());
+    Schema outputSchema = null;
```

Comment:
I don't think its worth having this small (3-4) line refactored into its own class.